collectors/gsuite/README.md

```diff
@@ -10,35 +10,57 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. G Suite Setup
+
+1. Create a Custom admin role. [Link](https://support.google.com/a/answer/1219251?hl=en&ref_topic=4514341)
+2. This admin role should have ‘Reports’ privilege.
+3. Create an admin user and assign him the role created above.
+4. Enable API access for G Suite. [Link](https://support.google.com/a/answer/60757?authuser=3)
+
+### 2. Google Cloud platform Setup
+
+1. Create a google cloud project [Link](https://console.cloud.google.com/home/dashboard)<br />
+   Note: Preferably create the cloud project with the same user as created in the previous section.
+2. Enable ‘Admin API’ [Link](https://console.cloud.google.com/apis/library/admin.googleapis.com)
+3. Create a service account [Link](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount)<br />
+   Note: Preferably download the JSON creds file.
+4. Delegating domain-wide authority to the service account [Link](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#delegatingauthority)<br />
+   Note: In Step 6 add scope as : https://www.googleapis.com/auth/admin.reports.audit.readonly
+5. Add app [Link](https://support.google.com/a/answer/45690?hl=en)    
```

Comment:
```From the Admin console Home page, go to Apps and then G Suite.
At the top right, click Add Services.
Find the Add Services link at the top right
Below the subscription you want to add, click Add It Now.
Follow the on-screen instructions to add the service to your organization's Google Account
```
I'd just replace `Add app` and the link with the above from the linked GSuite docs